apps/prairielearn/src/lib/question.js

```diff
@@ -1398,8 +1398,8 @@ module.exports = {
       // necessary for backward compatibility
       urls.calculationQuestionFileUrl = questionUrl + 'file';
 
-      // FIXME: broken?
-      urls.calculationQuestionGeneratedFileUrl = questionUrl + 'generatedFilesQuestion';
+      urls.calculationQuestionGeneratedFileUrl =
+        questionUrl + 'generatedFilesQuestion/variant/' + variant.id;
```

Comment:
Generated files must be associated with a variant; this URL was missing the variant ID.